lib/src/middleware/reselect_moved_dna_extensions.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:react/react.dart';
```

Comment:
let's rename this file to `reselect_moved_dna_extension_ends.dart` and the middleware function to `reselect_moved_dna_extension_ends_middleware`. Otherwise it's confusing, because extensions can also be selected, but that's not what this is doing; it's selecting the DNAends of those extensions.